### ISSUE TITLE
如果里面加载的是一个iframe,就不能拖动了

### ISSUE THREAD

**Issue by yurizhang** [jbaysolutions/vue-grid-layout#419]
![image](https://user-images.githubusercontent.com/20555332/79959635-f191cd00-84b6-11ea-927e-805c340a34be.png)

如上图, 左边加载的是一个iframe,就不能拖动了. 那个可以拖动的十字鼠标指针就没有了. 
右边的一个图按住空白地方可以拖动.

我尝试修改iframe z-index 好像也不行

**Comment by gmsa** [comment]
English only please

**Comment by ZhiJieRao** [comment]
> vue3+vite
> vue-grid-layout: 3.0.0-beta1

I also encountered the same problem, it is indeed impossible to drag, and the cross shaped mouse pointer for dragging is also gone. Setting z-index works for me.
This is how I solved it

Vue template pseudocode
`<grid-item>
    <div class="grid-item-wrapper">
	<iframe/>
    </div>
    <div class="grid-item-view"></div></grid-item>
`
CSS pseudocode
`
  .grid-item-wrapper{
	position: reactive;
	background-color: #fff;
	height: 100%;
	}
   .grid-item-view{
	position: absolute;
	with: 100%;
	height: 100%;
	z-index: 4;
	opacity: 0;
	}
`
Note: z-index only applies to sibling elements within the same parent container and requires the use of positional properties.
I hope my answer is useful
